dev-java/jtidy/jtidy-1.0-r1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-ant-2
+
+MY_PV="r938"
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="Java port of HTML Tidy, an HTML syntax checker and pretty printer"
+HOMEPAGE="http://jtidy.sourceforge.net/"
+SRC_URI="mirror://sourceforge/project/${PN}/JTidy/${MY_PV}/${PN}-${MY_PV}-sources.zip -> ${P}.zip"
+LICENSE="HTML-Tidy W3C"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+CDEPEND="dev-java/ant-core:0"
+
+DEPEND="
+	${CDEPEND}
+	>=virtual/jdk-1.8:*"
+
+RDEPEND="
+	${CDEPEND}
+	>=virtual/jre-1.8:*"
+
+BDEPEND="
+	app-arch/unzip
+	source? ( app-arch/zip )"
```

Comment:
i think this dependency is added automatically and is not needed anymore.

Reply:
> i think this dependency is added automatically and is not needed anymore.

Removed, and it still works ...